src/unibasicTemplates/setup.njk

```diff
@@ -3,7 +3,37 @@ subroutine mvom_setup(setupOptions)
 * Perform any needed environment setup
 
 com /S$MVOM/ S$MVOM.PROCESS
+com /S$MVOM/ S$MVOM.USER1, S$MVOM.USER2, S$MVOM.USER3, S$MVOM.USER4, S$MVOM.USER5
```

Comment:
You can declare on multiple lines like this?

Reply:
Yes.  I wasn't aware of it, but it works fine.  The only rule apparently is that the variable names are by ordinal position and not by name.  Each additional statement just adds more positions to the list.